### ISSUE TITLE
Cannot specify minVersion higher than 3.0 when using standalone

### ISSUE THREAD

**Issue by glenjamin** [Microsoft/dtslint#176]
This might also affect DefinitelyTyped, but I haven't tried in that context.

```
// TypeScript Version: 3.0
```
works

```
// TypeScript Version: 3.1
```
gives the error message
```
Error: Don't use `// TypeScript Version` for newer TS versions, use typesVerisons instead.
```

However `typesVersions` doesn't appear to affect the minimum version selection at all.

**Comment by JustinLivi** [comment]
In addition, the error message when running `dtslint` on expected 3.1 errors provides the following message:
```
ERROR: 27:37  expect  Compile error in typescript@3.0 but not in typescript@3.1.
Fix with a comment '// TypeScript Version: 3.1' just under the header.
```
Which, as @glenjamin noted, returns its own unsolvable error.

**Comment by sandersn** [comment]
This should be fixed now, by #182. It was not a good requirement even for DT.
